BLSQ/iaso
# POLIO-1118 fixes afro map

Multiple fixes in LQAS Afro map

⚠️ Please do not merge before Martin validate all those changes.

Related JIRA tickets : POLIO-1120, POLIO-1123, POLIO-1130, POLIO-1131, POLIO-1134
## Self proofreading checklist


## Changes

- use radio button to switch between `country` and `district`
- split load of maps 
- block drag on map while loading
- deep link center and zoom of maps 
- adapt legend while on `country` or `district` view
-  make shape borders thicker on district view

## How to test

Make sure you have some data on LQAS Afro map ( https://github.com/BLSQ/iaso/pull/690 -
Navigate to Poio/Lqas/map
Select 12 month in the select and  play with the maps.
Switch to district mode, zoom, reload the page, make sure both map are displaying the same thing after and before reload.

## Print screen / video

https://github.com/BLSQ/iaso/assets/12494624/b84e2ffa-a54b-45b9-a281-6514057d8e7e
